plugins/4.applicationInsights.ts

```diff
@@ -0,0 +1,83 @@
+// This plugin was inspired by https://github.com/nuxt-community/applicationinsights-module
```

Comment:
What's actually the reason for not using the original package ..?

Reply:
It doesn't work with Nuxt 3